dev/local-environment/configurations/partner-chains-setup/entrypoint.sh

```diff
@@ -55,7 +55,7 @@ echo "Permissioned candidates policy ID: $PERMISSIONED_CANDIDATES_POLICY_ID"
 echo "Setting values for NATIVE_TOKEN_POLICY_ID, NATIVE_TOKEN_ASSET_NAME, and ILLIQUID_SUPPLY_VALIDATOR_ADDRESS for chain-spec creation"
 export NATIVE_TOKEN_POLICY_ID="1fab25f376bc49a181d03a869ee8eaa3157a3a3d242a619ca7995b2b"
 export NATIVE_TOKEN_ASSET_NAME="52657761726420746f6b656e"
-export ILLIQUID_SUPPLY_VALIDATOR_ADDRESS="addr_test1wpy8ewg646rg4ce78nl3aassmkquf4wlxcaugqlxwzcylkca0q8v3"
+export ILLIQUID_SUPPLY_VALIDATOR_ADDRESS="addr_test1wq0mnff9xtddgxx8wjxt8sx68gxvf06yy2c60tqdgsp8ydsgvu7hq"
```

Comment:
Is it because genesis_utxo changed?

Reply:
yes

Reply:
Is it in `addresses.IlliquidCirculationSupplyValidator` field of addresses.json?